src/vuestic-theme/vuestic-components/va-slider/VaSlider.vue

```diff
@@ -1,40 +1,38 @@
 <template>
```

Comment:
![image](https://user-images.githubusercontent.com/5394573/58708695-affdbc80-83e2-11e9-91f5-c68ba3d82318.png)

Reply:
@nastassiadanilova any updates on that?